src/Elmish/Test.purs

```diff
@@ -94,13 +94,15 @@
 module Elmish.Test
   ( module Elmish.Test.Bootstrap
   , module Elmish.Test.Combinators
+  , module Elmish.Test.Discover
   , module Elmish.Test.Events
   , module Elmish.Test.Query
   , module Elmish.Test.SpinWait
   ) where
 
 import Elmish.Test.Bootstrap (testComponent, testElement)
 import Elmish.Test.Combinators (chain, chainM, forEach, mapEach, within, within', (##), ($$), (>>))
+import Elmish.Test.Discover (childAt, children, find, findAll, findFirst, findNth)
 import Elmish.Test.Events (change, click, clickOn, fireEvent)
-import Elmish.Test.Query (attr, exists, find, findAll, findFirst, findNth, html, prop, tagName, text)
+import Elmish.Test.Query (attr, count, exists, html, prop, tagName, text)
```

Comment:
I broke up the `Query` module into `Query` (reading properties of elements) and `Discover` (manipulating/traversing the DOM tree). Initially it was necessary for my implementation of `innerText` to avoid cyclic imports. I changed the implementation since then, but decided to leave the separation.